packages/runtime/container-runtime/src/compatUtils.ts

```diff
@@ -0,0 +1,200 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { FlushMode } from "@fluidframework/runtime-definitions/internal";
+import * as semver from "semver";
```

Comment:
node's semver implementation is surprisingly big. One thing you can do to help mitigate this is to only import the bit you need. Something like this should be possible:


```suggestion
import * as semverGte from "semver/functions/gte"
```

see https://www.npmjs.com/package/semver docs.

You should also double-check bundle impact of this change in case we might want to go with a more lightweight semver library which has all the features we need (IMO leaning into well-established one in the ecosystem is a totally reasonable call though, I'd squint harder at other libraries to make sure they're a good choice)

Reply:
importing `valid` and `lte` in basically the same way is fine and shouldn't generate much more code, they all use basically the same stuff.